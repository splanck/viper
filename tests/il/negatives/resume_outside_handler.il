il 0.1.2
func @main() -> void {
entry(%tok:ResumeTok):
  resume.next %tok
}
