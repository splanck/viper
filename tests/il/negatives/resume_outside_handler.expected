error: main:entry: resume.next %tok: resume.* only allowed in handler block
